lib/jarvis/commands/merge.rb

```diff
@@ -72,40 +72,42 @@ def execute
     Jarvis::Git.config(git, "user.email", committer_email)
     Jarvis::Git.config(git, "user.name", committer_name)
 
+    if branches.empty?
+      logger.info("No branches given, merging to PR's base branch: #{pull.base.ref}")
```

Comment:
I wonder if we should be more explicit and for the user to always specify which branch to land. I ask because there could be a case in which a base branch is 7.x, but we no longer desire to land this there, and we want to land it in master, but the person executing the Jarvis command did a typo, and the branch suddenly gets landed in 7.x instead of master.

That said, not a blocker.

Reply:
I think in those cases we should open a new PR that targets the right branch. Otherwise the PR will stay open since we're merging the commit to a different branch, and we no longer append the "closes #pr".

What I'm thinking is that we'll still to be able to execute a single merge command to a non-target branch. The use case is:
1. run `merge pr_url master 7.x`
2. realize it should go to 7.8 too, so run `merge pr_url 7.x`

For this I'd suggest renaming the old merge command to `backport` but remove the added comment to facilitate identifying the patch across branches.